tests/test_create_model.py

```diff
@@ -112,17 +138,6 @@ def test_create_model_must_not_reset_parent_namespace():
     ]
 
 
-def test_invalid_name():
-    with pytest.warns(RuntimeWarning):
-        model = create_model('FooModel', _foo=(str, ...))
-    assert len(model.model_fields) == 0
-
-
-def test_field_wrong_tuple():
-    with pytest.raises(errors.PydanticUserError):
-        create_model('FooModel', foo=(1, 2, 3))
```

Comment:
Why is this valid now? What would be the implication here?

Reply:
It's not, but this test was a duplicate.